src/client/components/TablePaginated/Filters/MultiSelect/hooks/useTooltipContent.ts

```diff
@@ -0,0 +1,58 @@
+import { useCallback, useMemo, useState } from 'react'
+
+import { Objects } from 'utils/objects'
+
+import { TablePaginatedFilterType } from 'meta/tablePaginated'
+
+import { useTablePaginatedFilterValue } from 'client/store/ui/tablePaginated/hooks'
+import { TablePaginatedFilter } from 'client/components/TablePaginated/types'
+
+type Props = {
+  fieldName: string
+  options: TablePaginatedFilter<TablePaginatedFilterType.MULTI_SELECT>['options']
+  path: string
+}
+
+type Returned = {
+  hideTooltip: () => void
+  showTooltip: () => void
+  tooltipContent: string | null
+}
+
+export const useTooltipContent = (props: Props): Returned => {
+  const { fieldName, options, path } = props
+  const filterValue = useTablePaginatedFilterValue<Array<string>>(path, fieldName)
+  const [canDisplayTooltip, setCanDisplayTooltip] = useState(true)
+
+  const valueToLabelMap = useMemo(() => {
+    return options.reduce((acc, { value, label }) => {
+      // eslint-disable-next-line no-param-reassign
+      acc[value] = label
```

Comment:
is it possible to add in eslint a rule where we can assign in certain functions eg reduce?